roles/restore/tasks/init_awx.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Get AWX object definition from pvc
+  community.kubernetes.k8s_exec:
+    namespace: "{{ tower_backup_pvc_namespace }}"
+    pod: "{{ meta.name }}-db-management"
+    command: >-
+      bash -c "cat '{{ tower_backup_dir }}/awx_object.yml'"
+  register: awx_object
+
+- name: Write temp AWX definition template file
```

Comment:
I don't think this step is necessary. You could just read the YAML into a variable and pass that directly to the `definition` parameter of the `k8s` module.

Reply:
That's what I was originally doing, but the backed up YAML files are actually templates.  From what I read and from trying it, it looks like you can only pass a file path to `template:`, not a string.